### PR TITLE
added missing css for the leaflet map tooltip

### PR DESCRIPTION
**Pull request by hkernbach**
added last commit from already merged PR (3.4):
https://github.com/arangodb/arangodb/pull/6371

### CODE REVIEW

**Comment by Simran-B** [comment]
Is the change to `Documentation/DocuBlocks/Rest/Graph/general_graph_edge_definition_remove_http_examples.md` intended to be in this PR?

**Comment by hkernbach** [comment]
@Simran-B Ups, originally not. But it is fine. It will not break anything. New C++ Gharial API is included in 3.4, so the change is correct. (even when it is misplaced)

File: Documentation/DocuBlocks/Rest/Graph/general_graph_edge_definition_remove_http_examples.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The name of the edge collection used in the definition.
 
 @RESTQUERYPARAMETERS
 
-@RESTQUERYPARAM{dropCollection,boolean,optional}
+@RESTQUERYPARAM{dropCollections,boolean,optional}
```
Comment (mchacki) [inline Documentation/DocuBlocks/Rest/Graph/general_graph_edge_definition_remove_http_examples.md]: Can we please remove this change from this PR?
Unreleated and contained somewhere else.

File: tests/js/client/http/api-gharial-spec.js [hunk 2]
```diff
@@ -846,4 +846,37 @@ describe('_api/gharial testing cross-graph deletes', () => {
 
     expect(deleted).to.equal(true);
   });
+
+  it('should not create multiple orphan collection entries', () => {
```
Comment (mchacki) [inline tests/js/client/http/api-gharial-spec.js]: Can we remove this test from this PR?
It is contained in a different PR and is totally unreleated.

**Pull request closed without merging**